.github/workflows/ci-cd.yaml

```diff
@@ -25,6 +25,7 @@ jobs:
         env:
           EMAIL: releases@foxygo.at
           GIT_COMMITTER_NAME: Foxygoat Releases
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
`GH_TOKEN` env var not `GITHUB_TOKEN`: See https://github.com/foxygoat/howl/actions/runs/7541299915/job/20527732490

```
gh: To use GitHub CLI in a GitHub Actions workflow, set the GH_TOKEN environment variable. Example:
  env:
    GH_TOKEN: ${{ github.token }}
```

Reply:
thank you - addressed.